<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOutDataConfig_UI</class>
 <widget class="QDialog" name="DialogOutDataConfig_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>177</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>177</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>177</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>参数导出</string>
  </property>
  <widget class="QPushButton" name="pushButtonSelectParams">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择要导出的参数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelectDir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择导出目录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDir">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSelectedParams">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>301</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutData">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonMutiFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>每个参数一个文件</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonSingleFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>全部参数导出为一个文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
